<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RefereeView</class>
 <widget class="QWidget" name="RefereeView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Races:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Participants:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QTableWidget" name="participantsTable"/>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="closeRaceButton">
     <property name="text">
      <string>Close Race</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" rowspan="2">
    <widget class="QTableWidget" name="racesTable"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
